.github/workflows/llm_performance_tests.yml

```diff
@@ -84,6 +84,42 @@ jobs:
           # hide time info
           sed -i 's/str(end - st)/"xxxxxx"/g' run.py
           python run.py
+          current_date=$(date +%Y-%m-%d)
+          for file in *.csv; do
+              if [ -e "$file" ] && [[ "$file" == *"$current_date"* ]]; then
+                  filename="${file%.*}"
+                  extension="${file##*.}"
+
+                  new_filename="${filename}_test1.${extension}"
+                  mv "$file" "$new_filename"
+              fi
+          done
```

Comment:
just use sed to change the file name in run.py

Reply:
ok